frontend/app/dashboard/template.hbs

```diff
@@ -95,7 +95,7 @@
         {{/card.header}}
         {{#card.body}}
           <p>
-            Adfinis SyGroup AG<br>
```

Comment:
Is the AG also removed from the official name?

Reply:
Dont know but the current name also doesn't have AG in it

Reply:
So do we leave it as is or change it to AG?

Reply:
Ill ask

Reply:
Changed it to AG